Source/Pawnmorphs/Esoteria/PawnmorphPatches.cs

```diff
@@ -275,7 +275,9 @@ private static void MassPatchFormerHumanChecks([NotNull] Harmony harmonyInstance
             //incidents 
             PatchIncidents(methodsToPatch);
 
-            //interaction patch 
+            //interaction patch
+            methodsToPatch.Add(typeof(HealthCardUtility).GetMethod("CreateSurgeryBill", STATIC_FLAGS));
+            methodsToPatch.Add(AccessTools.Method("RimWorld.FloatMenuMakerMap+<>c__DisplayClass9_7:<AddHumanlikeOrders>b__5"));
```

Comment:
We don’t want to look for the names of compiler generated functions as they can change any update to something completely different. We’ll have to do more complicated reflection to examine the parameters, and return type as these are less likely to change

Reply:
My reasoning was that it would only ever change if they changed the function generating them, and in that case we would need to re-evaluate the patch anyway.
But I may underestimate how likely an anonymous method is to changing compared to a defined method.
So I can try. We may end up running the transpiring across multiple seemingly identical methods to ensure we hit the right one though, however that might not be a problem.